setup

```diff
@@ -243,9 +243,9 @@ if [[ $BTK_WALLET == 1 ]]; then
 		sleep 5
 
 		# Generate some blocks, and get their hashes/merkle roots
-		docker-compose exec bitcoind-regtest bitcoin-cli getblock 0 > ./Electron-Cash-SLP/genesis.txt
-		docker-compose exec bitcoind-regtest bitcoin-cli getblock 150 > ./Electron-Cash-SLP/fork.txt
-		docker-compose exec bitcoind-regtest bitcoin-cli getblock 200 > ./Electron-Cash-SLP/verification.txt
+		docker-compose exec -T bitcoind-regtest bitcoin-cli getblock 0 > ./Electron-Cash-SLP/genesis.txt
+		docker-compose exec -T bitcoind-regtest bitcoin-cli getblock 150 > ./Electron-Cash-SLP/fork.txt
+		docker-compose exec -T bitcoind-regtest bitcoin-cli getblock 200 > ./Electron-Cash-SLP/verification.txt
```

Comment:
-T disables pseudo-tty allocation. By default `docker-compose exec` allocates a TTY.  
TTY is not supported by CI services.